### DISCUSSION TITLE
Solution not moving

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#290] [General]
Hi,

What does this sentence mean?
Solution time 9e-08 less than 1e-6, not moved

Does it mean the shape in not optimizing anymore, and we should stop the simulation, or it means that the solution does not converge?

Thank you
regards,
Hossein

**Comment by friedenhe** [comment]
No, this is not an error or warning. We rename the solution folder from CFD, like 2000, to a folder name like 0.000001. This message just says that the latest solution folder is 9e-8 and we do not rename it.
